config.pkg/guard-rake.yaml

```diff
@@ -0,0 +1,3 @@
+depends:
+  - guard
```

Comment:
Looking at the list of dependencies for this gem https://rubygems.org/gems/guard-rake I see that both `guard` and `rake` gems are already listed as deps. Quarry takes care of the gem dependencies already, no need to have it in the configuration file. This file can be removed from the PR.

Reply:
Good to know, thanks. Removed.